### ISSUE TITLE
Wrong GPIO names in README

### ISSUE THREAD

**Issue by Kitsok** [openbmc/x86-power-control#4]
Hello!

The README.md indicates that the following lines must be defined in DTS:
RESET_BUTTON RESET_OUT POWER_BUTTON POWER_OUT

But actually the following list of GPIO names is used in the application:
RESET_BUTTON
RESET_OUT
POWER_BUTTON
POWER_OUT
PS_PWROK
SIO_POWER_GOOD
SIO_ONCONTROL
SIO_S5
NMI_BUTTON
NMI_OUT
ID_BUTTON
POST_COMPLETE

Could you please fix the README file?

**Comment by brandonkimbk** [comment]
Yes, this would be great as this daemon also seems to be lacking description of these critical GPIOs
